### ISSUE TITLE
Only 1 image is shown in MultiImageProvider and cannot navigate to the next image

### ISSUE THREAD

**Issue by keanallen** [thesmythgroup/easy_image_viewer#11]
## Describe the Issue

This does not work in Desktop (Windows) at Flutter `version 2.10.3 channel stable`

Todo

## Expected behavior

I should be able to see all the images declared in `MultiImageProvider`

Todo

## Steps to Reproduce

Just try it on Desktop

n/a

## Other Information

n/a

**Comment by jfahrenkrug** [comment]
@keanallen Thank you for reporting this. I haven't been able to try it on Windows, but I think these links are relevant to the issue:
https://stackoverflow.com/questions/69424933/flutter-pageview-not-swipeable-on-web-desktop-mode
https://docs.flutter.dev/release/breaking-changes/default-scroll-behavior-drag
https://www.reddit.com/r/flutterhelp/comments/thyl47/how_to_use_arrow_keys_or_mouse_swipe_for_pageview/

Basically the default behavior is that a mouse can't scroll a PageView. I'll look into this soon, and I'll appreciate your help testing it on Windows once I have a fix. Thanks!

**Comment by keanallen** [comment]
> @keanallen Thank you for reporting this. I haven't been able to try it on Windows, but I think these links are relevant to the issue: https://stackoverflow.com/questions/69424933/flutter-pageview-not-swipeable-on-web-desktop-mode https://docs.flutter.dev/release/breaking-changes/default-scroll-behavior-drag https://www.reddit.com/r/flutterhelp/comments/thyl47/how_to_use_arrow_keys_or_mouse_swipe_for_pageview/
> 
> Basically the default behavior is that a mouse can't scroll a PageView. I'll look into this soon, and I'll appreciate your help testing it on Windows once I have a fix. Thanks!

Sure. Im using this in our desktop chat application. Waiting for the fix <3
![image](https://user-images.githubusercontent.com/45480782/160026524-22c6ff88-a8a7-4777-9662-3eb517f6e78a.png)

**Comment by jfahrenkrug** [comment]
@keanallen In your use case, what behavior would you expect? It sounds like "< Prev" and "Next >" buttons would be the most intuitive? Or would you expect the user to click and drag the image to advance to the next image?

**Comment by jfahrenkrug** [comment]
@keanallen PR to enable dragging. Feel free to give it a try on Windows: https://github.com/thesmythgroup/easy_image_viewer/pull/13

**Comment by jfahrenkrug** [comment]
I'm closing this, feel free to reopen if needed.

**Comment by henry2man** [comment]
Hi there. I'm also using the package on Web/MacOS but I cannot change pages using mouse and/or using "drag" gestures with the mouse.

Also if I try to scroll horizontally (with an MX Master 3) I cannot change page, even if I try as fast as I can:


https://user-images.githubusercontent.com/4610108/181217239-8c1b08fa-706d-48df-8a2e-bf3ea2ecb90f.mp4



Probably this should be reopened. And having some `<PREV` & `NEXT>` buttons could be great, even for mobile (when a gallery is loaded, there isn't any visual indicative about other images).
